mocks/router/middlewares/JSONHandler.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.14.0. DO NOT EDIT.
+// Code generated by mockery v2.20.0. DO NOT EDIT.
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.